kuma/api/v1/views.py

```diff
@@ -245,6 +245,12 @@ def search(request, locale):
                           Q('term', locale={'value': locale, 'boost': 8}),
                       ])))
 
+    # Add excerpts with search results highlighted
+    search = search.highlight(*WikiDocumentType.excerpt_fields)
```

Comment:
I suppose it's outside the scope to have "title" added to that. I guess because of how it's indexed in ES.